src/Eurofurence.App.Server.Web/Startup.cs

```diff
@@ -173,7 +173,7 @@ public void ConfigureServices(IServiceCollection services)
 
             services.Configure<LoggerFilterOptions>(options => { options.MinLevel = LogLevel.Trace; });
 
-            services.AddDbContextPool<AppDbContext>(options =>
+            services.AddDbContext<AppDbContext>(options =>
```

Comment:
Why would you replace the DbContextPool with a single DbContext? We can expect a _very_ high number of concurrent requests once the apps go live, so a proper pool sounds like the better solution.

Reply:
After testing with the pool enabled again, I didn't notice any problems with the private message endpoint, so re-enabling it